docs/how_to_make.md

```diff
@@ -0,0 +1,87 @@
+for free. Unzip the package to any location of your choice. Remember this
+location, because this will be the `Arduino SDK`, everytime we refer to it.
+Open the `arduino` executable inside the SDK folder to run the IDE.
+
+2. Copy all the folders inside the `libraries` folder in this repo into the
+`libraries` folder of your Arduino SDK. These are dependencies the WatchDuino
+code needs to compile.
+
+3. On the Arduino IDE, click on `File->Open`. Navigate to where you cloned
+the WatchDuino repository, and pick the `watch/watch.ino` file to open.
+All of the WatchDuino files should now have been loaded in the IDE.
+
+4. On the IDE now go to `Tools->Board` and pick `Arduino Pro or Pro Mini`.
+Then pick `Processor:ATMega328 (3.3V, 8Mhz)`. This step is needed because the
+IDE needs to know the microcontroller that we are using, in order to apply
+the correct compilation options.
+
+5. After all this, you should be able to compile the code. On the IDE, click
+on the `Verify` button (the first icon in the toolbar, showing a tick). After a
+minute it should be done without errors.
+
+6. You are ready now to upload the code into your Arduino Mini. Connect
+the Arduino to your PC via USB. Then go to `Tools->Port`, and you should
+be able to see something in there (in my case it's '/dev/ttyUSB0', but this
+will likely be different depending on your OS. It's essentially an identifier
+for your USB port where your Arduino is plugged). If you see `Tools->Port`
+greyed out, it's because you are not running the IDE with sufficient
+permissions. If you are on Linux try running the program with a superuser
+(eg. `sudo` on Ubuntu).
+
+7. After that, click on the `Upload` button (the second icon in the toolbar,
+showing a right pointing arrow). The Arduino Mini should now be loaded with
+the WatchDuino firmware.
+
+## Mounting a prototype on a breadboard
+
+Before you go and build the real thing, it's advisable that you build a
+prototype version on a breadboard. This way you can test that the code
+is correctly loaded into the Arduino board, and that you understood
+all the connections in the schematic right.
+
+The schematic can be found in [the `schema` folder of this repo](../schema),
+you should see an `.fzz` file in there. You can use the
+[Fritzing software](http://fritzing.org/download/?donation=0) to open this
+file, and it'll show you a detailed view of the components connected together,
+which you can zoom and hightlight the connections in.
+
+After you've successfully flashed the Arduino with the firmware, leave it
+connected to the USB, and place it on the breadboard. Then:
+
+- Place the buzzer. You should hear a "beep" if you reset the Arduino.
+- Place the screen. You should see the WatchDuino interface if you reset
+the Arduino.
+- Place the buttons. You should be able to use them to navigate through
+the menus.
+- Place the bluetooth module. You shoud see an intermitent led flashing.
```

Comment:
You will see a led if you are using the [full bluetooth module for arduino](http://img.alibaba.com/img/pb/893/290/738/738290893_793.jpg) if you are using just the hm10(http://thumbs1.ebaystatic.com/d/l225/m/mEapdoSJYYG2mZngxgIe97w.jpg) there will be no led